interpreter/cling/lib/Utils/Validation.cpp

```diff
@@ -18,12 +19,41 @@
 # include <unistd.h>
 #endif
 
-
+#define CACHESIZE 8
```

Comment:
I'm taking care of this - I'll discuss my changes with Cristina tomorrow. (E.g. here I suggest `std::array`.) Thanks nonetheless, Philippe!